frontend/packages/dev-console/src/components/progressive-list/ProgressiveList.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import FormSectionDivider from '../import/section/FormSectionDivider';
+import { getGeneratedText } from './progressive-list-utils';
+import './ProgressiveList.scss';
+
+interface ProgressiveListProps {
+  visibleItems: string[];
+  onVisibleItemsChanged: (item: string) => void;
+  text: string;
+}
+
+const ProgressiveList: React.FC<ProgressiveListProps> = ({
+  visibleItems,
+  children,
+  onVisibleItemsChanged,
+  text,
+}) => {
+  const items = [];
+  return (
+    <React.Fragment>
+      {React.Children.map(children, (child: React.ReactElement, index: number) => {
```

Comment:
Since child could be anything, please check to ensure it's not null and that `child.type  === ProgressiveListItem`.... I think this should work. If not, return null so not render it.